src/main/java/com/amazonaws/services/sqs/AmazonSQSIdleQueueDeletingClient.java

```diff
@@ -70,10 +70,10 @@
     public static final String IDLE_QUEUE_RETENTION_PERIOD = "IdleQueueRetentionPeriodSeconds";
     public static final long MINIMUM_IDLE_QUEUE_RETENTION_PERIOD_SECONDS = 1;
     public static final long MAXIMUM_IDLE_QUEUE_RETENTION_PERIOD_SECONDS = TimeUnit.MINUTES.toSeconds(5);
-    
+    public static final long HEARTBEAT_INTERVAL_SECONDS_DEFAULT = 5;
+    public static final long HEARTBEAT_INTERVAL_SECONDS_MIN_VALUE = 0;
```

Comment:
I would make this 1 and change the comparison from a `<=` to a '<'. Since this is a discrete value I think making the "minimum" a valid value is easier to understand.

Reply:
Changed the value to 1 as well as the comparison to '<'.